pkgs/gnolang/op_eval.go

```diff
@@ -64,6 +64,8 @@ func (m *Machine) doOpEval() {
 						x.Value))
 				}
 			} else {
+				x.Value = strings.ReplaceAll(x.Value, "_", "")
```

Comment:
Change this to be also for binary/octals/hexadecimals? (so outside of the if/else).

Reasoning for this stems from the [Integer literals](https://go.dev/ref/spec#Integer_literals) section. The underscores can be in all number literals, not just decimals.